ci/prow-entrypoint.sh

```diff
@@ -316,13 +321,13 @@ main() {
             ;;
         "rhcos-9-build-test-qemu")
             setup_user
-            cosa_init "ocp-rhel-9.4"
+            cosa_init "ocp-rhel-9.6"
```

Comment:
We can't change our primary CI over to 9.6 yet. The master branch currently tracks 4.18, which will ship on 9.4. I guess we could add another CI run for it for now, though it's going to be mostly covered by SCOS anyway, at least until branching on the RHEL side.